wagtailmenus/models.py

```diff
@@ -106,6 +106,12 @@ class MenuItem(models.Model):
         blank=True,
         help_text=_("Must be set if you wish to link to a custom URL."),
     )
+    link_handle = models.SlugField(
```

Comment:
I agree on both, will change it. Wasn't sure about the design considerations of the link_* attributes and the priming.